<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forensic Analyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbl_params">
        <property name="text">
         <string>Forensic Analyzer Parameters:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="formWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QFormLayout" name="formLayout_1">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_input">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Input Log Files:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTextBrowser" name="txt_input">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_config">
           <property name="text">
            <string>Config Files:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextBrowser" name="txt_config">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_fixed">
           <property name="text">
            <string>Fixed &amp; Time:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextBrowser" name="txt_fixed">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_optional">
           <property name="text">
            <string>Optional Params:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lnE_optional">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>--intevents</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_btn">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_startAnalyzing">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>START ANALYZING</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/resources/res/icons/StartAnalysis.png</normalon>
         </iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="txt_sec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menuInput_Log_Files">
    <property name="title">
     <string>Input Log Files</string>
    </property>
    <widget class="QMenu" name="menuModify_Log">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Modify Log</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/resources/res/icons/LogFileModify.png</normalon>
      </iconset>
     </property>
    </widget>
    <widget class="QMenu" name="menuDelete_Log">
     <property name="title">
      <string>Delete Log</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/resources/res/icons/LogFileDelete.png</normalon>
      </iconset>
     </property>
    </widget>
    <addaction name="actionAdd_Log_File"/>
    <addaction name="menuModify_Log"/>
    <addaction name="menuDelete_Log"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration_Files">
    <property name="title">
     <string>Configuration Files</string>
    </property>
    <widget class="QMenu" name="menuModify_Config">
     <property name="title">
      <string>Modify Config</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/resources/res/icons/ConfigFileModify.png</normalon>
      </iconset>
     </property>
    </widget>
    <widget class="QMenu" name="menuDelete_Config">
     <property name="title">
      <string>Delete Config</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/resources/res/icons/ConfigFileDelete.png</normalon>
      </iconset>
     </property>
    </widget>
    <addaction name="actionAdd_Config_File"/>
    <addaction name="menuModify_Config"/>
    <addaction name="menuDelete_Config"/>
   </widget>
   <widget class="QMenu" name="menuTime_Filter">
    <property name="title">
     <string>Time Filter</string>
    </property>
    <addaction name="actionSet_Time_From"/>
    <addaction name="actionSet_Time_To"/>
    <addaction name="actionClear_All_Filters"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUsage"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebug_Level">
    <property name="title">
     <string>Debug Level</string>
    </property>
    <addaction name="action1_Critical"/>
    <addaction name="action2_Error"/>
    <addaction name="action3_Warning"/>
    <addaction name="action4_Notification"/>
    <addaction name="action5_Informative"/>
    <addaction name="action6_Debug"/>
   </widget>
   <addaction name="menuInput_Log_Files"/>
   <addaction name="menuConfiguration_Files"/>
   <addaction name="menuTime_Filter"/>
   <addaction name="menuDebug_Level"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Log_File"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Config_File"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Time_From"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Time_To"/>
   <addaction name="separator"/>
   <addaction name="actionClear_All_Filters"/>
   <addaction name="separator"/>
   <addaction name="action1_Critical"/>
   <addaction name="separator"/>
   <addaction name="action3_Warning"/>
   <addaction name="separator"/>
   <addaction name="action6_Debug"/>
   <addaction name="separator"/>
   <addaction name="actionUsage"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Log_File">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/LogFileAdd.png</normalon>
     <activeon>:/resources/res/icons/LogFileAdd.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>Add Log File</string>
   </property>
   <property name="toolTip">
    <string>Add input log file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAdd_Config_File">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/ConfigFileNew.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add Config File</string>
   </property>
   <property name="toolTip">
    <string>Add configuration file, where correlation rules will be placed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSet_Time_From">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/TimeFilter.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Set Time From</string>
   </property>
   <property name="toolTip">
    <string>Filter only event from this time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+&gt;</string>
   </property>
  </action>
  <action name="actionSet_Time_To">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/TimeFilter.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Set Time To</string>
   </property>
   <property name="toolTip">
    <string>Filter only event to this time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+&lt;</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/Help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Usage</string>
   </property>
   <property name="toolTip">
    <string>Show Usage</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action1_Critical">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>1 - Critical</string>
   </property>
   <property name="toolTip">
    <string>1 - Critical Messages Only (MIN)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action2_Error">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>2 - Error</string>
   </property>
   <property name="toolTip">
    <string>2 - Error Messages and Above</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action3_Warning">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>3 - Warning</string>
   </property>
   <property name="toolTip">
    <string>3 - Warning Messages and Above (DEFAULT)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action4_Notification">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>4 - Notification</string>
   </property>
   <property name="toolTip">
    <string>4 - Notification Messages and Above</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="action5_Informative">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>5 - Informative</string>
   </property>
   <property name="toolTip">
    <string>5 - Informative Messages and Above</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="action6_Debug">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/DebugLevel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>6 - Debug</string>
   </property>
   <property name="toolTip">
    <string>6 - Debug Messages (MAX)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionClear_All_Filters">
   <property name="icon">
    <iconset>
     <normalon>:/resources/res/icons/ClearFilters.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear All Filters</string>
   </property>
   <property name="toolTip">
    <string>Clear All Time Filters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lnE_optional</tabstop>
  <tabstop>txt_sec</tabstop>
  <tabstop>txt_input</tabstop>
  <tabstop>txt_config</tabstop>
  <tabstop>txt_fixed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
